examples/.gitignore

```diff
@@ -1,3 +1,4 @@
 _site
 .sass-cache
 .jekyll-metadata
+Gemfile.lock
```

Comment:
Is it bad practice to check in a `Gemfile.lock`?

Reply:
You are alright! Assuming I am not with a rubygem, Gemfile.lock should be in the repository. It's used as a snapshot of all your required gems and their dependencies.